super-intel.c:5063:2: error: right-hand operand of comma expression has no effect [-Werror=unused-value]
  snprintf((char *) spare->sig, MAX_SIGNATURE_LENGTH,
  ^
cc1: all warnings being treated as errors
make: *** [super-intel.o] Error 1

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: mdadm-4.0/super-intel.c
===================================================================
--- mdadm-4.0.orig/super-intel.c
+++ mdadm-4.0/super-intel.c
@@ -5725,7 +5725,7 @@ static int write_super_imsm_spares(struc
 	spare->cache_size = mpb->cache_size;
 	spare->pwr_cycle_count = __cpu_to_le32(1);
 
-	snprintf((char *) spare->sig, MAX_SIGNATURE_LENGTH,
+	(void)snprintf((char *) spare->sig, MAX_SIGNATURE_LENGTH,
 		 MPB_SIGNATURE MPB_VERSION_RAID0);
 
 	for (d = super->disks; d; d = d->next) {
